### DISCUSSION TITLE
How to preserve plain html

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#5071] [Q&A] [answered]
I have a lot of R functions that create plain html output.  When a function produces a table, the plain html it uses has very little wasted vertical space, but when rendering with `Quarto` I get the standard format with striped rows, etc.   Is there a way to tell `Quarto` that a section of included html is to be kept as-is?

**Comment by cscheid** [comment, marked as answer]
https://quarto.org/docs/prerelease/1.3/tables.html

Specifically, use `<table data-quarto-disable-processing="true">...</table>`

**Reply by harrelfe** [reply]
Thanks a bunch.  Is there something comparable for non-table elements?  E.g. with `Quarto` in action I get taller vertical spaces between tables and other text.

**Reply by cscheid** [reply]
Have you confirmed if the HTML is being changed from what you have to the resulting file? I assume the changes you see are coming from our CSS and not from quarto not preserving the HTML itself. (In which case, the fix needs to be through HTML itself as well.) If not, can you post a full example of the problem you're seeing? Thanks.

**Reply by mcanouil** [reply]
FYI, for `gt` tables (as of `v0.9.0`), new options to `gt::tab_options()` allow to disable Quarto processing easily.
See <https://gt.rstudio.com/reference/tab_options.html>.
![image](https://user-images.githubusercontent.com/8896044/229430743-eb0aa8aa-c345-40f4-850f-6ce739e335cf.jpeg)

**Reply by harrelfe** [reply]
Sorry for the late reply.  The `disable-processing` directive did just what I need, and the new `gt` option is also much appreciated.
